images/daemon/pkg/daemon/kata_openshift.go

```diff
@@ -354,6 +355,11 @@ func uninstallRPMs(k *KataOpenShift) error {
 }
 
 func installRPMs(k *KataOpenShift) error {
+	var (
+		sourceCtx 	*types.SystemContext
+		payloadImage 	string
+	)
```

Comment:
Since these are not used till line 389, maybe you could skip the definition here, and define them where they are used?

Reply:
I'm used to defining variables at the beginning of a function. Is it common go coding style to do it differently?

Reply:
My comment was about the location but rather the duplication. For example, you are redefining the variable with ":=" [here](https://github.com/openshift/kata-operator/pull/51/files#diff-2c9b50b98430081b2bae1f8eed978d7b21c1d24e1c30cd754063c209a7a5c405R390) again.

Reply:
now I get it :-) yes that makes sense then. Thanks!

Reply:
> I'm used to defining variables at the beginning of a function. Is it common go coding style to do it differently?

I'm not sure about it.  However in this func since you are using the two vars much later, it might make sense to use the short var declaration approach for better readability. Wdyt ?

Reply:
got rid of the var block and declared the variables directly with :=